### ISSUE TITLE
Events recurring on multiple days are not treated correctly

### ISSUE THREAD

**Issue by michaelcadilhac** [ical2org-py/ical2org.py#39]
Consider this:
```
BEGIN:VCALENDAR
VERSION:2.0
CALSCALE:GREGORIAN
X-WR-CALNAME:Self
X-WR-TIMEZONE:America/Chicago

BEGIN:VEVENT
DTSTART;TZID=America/Chicago:20190912T101000
DTEND;TZID=America/Chicago:20190912T132000
RRULE:FREQ=WEEKLY;WKST=MO;UNTIL=20191122T055959Z;BYDAY=TH,TU
DTSTAMP:20190909T202045Z
CREATED:20190829T232009Z
DESCRIPTION:
SEQUENCE:0
STATUS:CONFIRMED
SUMMARY:Test
END:VEVENT

END:VCALENDAR
```
The resulting org file has events recurring only on Thursdays, not on Tuesdays.

Thanks for that great script!

**Comment by asoroa** [comment]
Hi,
I think this is the same as pull request #35. I haven't merge it because I'm planing to integrate `dateutil.rrule` into ical2orgpy, but never get to have the time to do it. You can try merging #35 as a  quick fix.

**Comment by spookylukey** [comment]
This is fixed in d71d06e665e3bad24459de3e8fd5dac353910ebe now.
